repose-aggregator/commons/commons-test/src/main/scala/org/openrepose/commons/test/ConfigValidator.scala

```diff
@@ -56,7 +56,7 @@ object ConfigValidator {
       classOf[ConfigValidator].getClassLoader)
     factory.setFeature("http://apache.org/xml/features/validation/cta-full-xpath-checking", true)
     new ConfigValidator(factory
-      .newSchema(schemas.map(_.openStream).map(new StreamSource(_)).toArray[Source])
+      .newSchema(schemas.map(u => new StreamSource(u.toString)).toArray[Source])
```

Comment:
I suppose it's somewhat of preference thing, but I prefer the way this was being done where each transformation is in its own `map` call. So:
```
.newSchema(schemas.map(_.toString).map(new StreamSource(_))).toArray[Source])
```

I think that ends up being one character longer, but for whatever reason, it seems clearer and cleaner to me. Not that it is hard to understand as-is, by any means.

Reply:
I actually find it less clear with an added layer, besides physically creating a new object you are also mentally creating another object that has to be tracked.

Reply:
Fair enough.